cypress/integration/index.spec.js

```diff
@@ -12,7 +12,7 @@ describe('uploadcare widget', () => {
 
     cy.get('.uploadcare--menu__item_tab_url').click()
     cy.get('.uploadcare--input').type(
-      'https://ucarecdn.com/3db26848-d1c6-4b3d-8cfa-13cc5a76f2a1/small.png{enter}'
+      'https://ucarecdn.com/e8701017-071b-42f4-b4db-0592b24931f9/small.png{enter}'
```

Comment:
Dima was using uuid from own personal project, and it's gone. I've uploaded it to the shared UC project.